parameters.json.example

```diff
@@ -0,0 +1,7 @@
+{
+  "url": "https://sgwdev.behoerden-serviceportal.de",
+  "user": "myUser",
+  "duplicateProcesskeyAction": "ERROR",
+  "deploymentName": "ozghub-process-template",
```

Comment:
Ich glaube, wenn wir das als Default lassen, sollten wir in der Anleitung nochmal explizit schreiben, dass der Wert angepasst werden muss, da wir sonst wahrscheinlich viele Deployments mit dem Namen `ozghub-process-template` haben (obwohl es sich bereits um einen angepassten Prozess handelt).

Alternative Idee: Ich nutze momentan für **alle** Prozesse den Deploymentnamen `DeploymentFromLocalRepository`. Daraus kommt dann keine Assoziation mit dem Prozessnamen (braucht es ja auch nicht, sieht man an der Prozess-ID). Wäre das vielleicht auch geeignet?